doc/verification.rst

```diff
@@ -30,11 +30,13 @@ Please note that this work is still working in progress.
 Getting Started
 ---------------
 
-Prerequisites
-~~~~~~~~~~~~~
+Prerequisites & Environment Setup
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
--  VCS RTL simulator (need to support UVM 1.2)
--  `Setup the RISC-V instruction generator and ISS sim environment <https://github.com/google/riscv-dv#getting-started>`_
+- VCS RTL simulator (needed to support UVM 1.2)
+- RISCV-DV Prerequisites - https://github.com/google/riscv-dv#prerequisites
+- GCC setup - https://github.com/google/riscv-dv#compile-generated-programs-with-gcc
+- ISS setup - https://github.com/google/riscv-dv#run-iss-instruction-set-simulator-simulation note that commit log can be enabled in spike by passing ``--enable-commitlog`` to the configure script.
```

Comment:
small nit: maybe add a dash between the note about enable-commitlog and the iss setup link, something like this:
`https://github.com/google/riscv-dv#run-iss-instruction-set-simulator-simulation - Note that...`

I think this is a useful change to the documentation, thank you for adding it @GregAC!